frontend/packages/shared/src/components/ScrollToTop/ScrollToTop.tsx

```diff
@@ -1,18 +1,12 @@
-import { History } from 'history';
-import { useContext, useLayoutEffect } from 'react';
-import { UNSAFE_NavigationContext } from 'react-router-dom';
+import { useLayoutEffect } from 'react';
+import { useLocation } from 'react-router-dom';
 
 export const ScrollToTop = () => {
-  const navigator = useContext(UNSAFE_NavigationContext).navigator as History;
-  useLayoutEffect(() => {
-    const unlisten = navigator.listen(({ action }) => {
-      if (action !== 'POP') {
-        window.scrollTo({ left: 0, top: 0 });
-      }
-    });
+  const location = useLocation();
 
-    return unlisten;
-  }, [navigator]);
+  useLayoutEffect(() => {
+    window.scrollTo({ left: 0, top: 0 });
+  }, [location]);
```

Comment:
Muuttuuko tän ScrollToTopin toiminta jotenkin oleellisesti tämän muutoksen myötä? Jos oikein käsitin, niin erot ovat seuraavat:
- vanha implementaatio skrollasi ylös vanhan sivun unmountin yhteydessä
- ... paitsi jos unmountin aiheutti paluu taaksepäin selaimen historiassa? 
- uusi implementaatio skrollaa ylös uuden sivun mountin yhteydessä

Reply:
Hyvä huomio! Täytyykin vielä perehtyä tähän vähän paremmin.

Reply:
Sain nyt ehkä toimimaan oikein